modules/portmaster/src/app/widgets/prompt-widget/prompt-widget.scss

```diff
@@ -4,7 +4,7 @@
   display       : flex;
   flex-direction: column;
 
-  padding-top: 1px;
+  padding-top   : 1px;
```

Comment:
We should definitely stick to one format. Let's talk about that next week.

Reply:
I already talked to like who now also has the auto formatter installed ;)